lib/afinter.c

```diff
@@ -408,9 +408,8 @@ afinter_postpone_mark(gint mark_freq)
 {
   if (mark_freq > 0)
     {
-      iv_validate_now();
       g_static_mutex_lock(&internal_mark_target_lock);
-      next_mark_target = iv_now;
+      clock_gettime(CLOCK_MONOTONIC, &next_mark_target);
       next_mark_target.tv_sec += mark_freq;
```

Comment:
Another way to avoid the actual problem would be to call `iv_init()` and `iv_deinit()` in the source thread, but we don't use ivykis at all in `LogThreadedSourceDriver`. This would be weird, but it is maybe better than reimplementing the same time cache that ivykis has.